models/Drinks.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const DrinksSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    unique: true,
+  },
+  type: {
+    type: String,
+    enum: ["coffee", "tea", "beer", "wine"],
+  },
+  tags: [{ type: String }],
```

Comment:
Still a little confused about the whole purpose of the tags. Maybe you can fill me in again now that I see the schema.